x-pack/plugins/monitoring/server/lib/alerts/fetch_cpu_usage_node_stats.ts

```diff
@@ -179,9 +179,7 @@ async function fetchContainerStats(
       }
 
       const limitsNotSet = node.quota_micros_max.value === -1 && node.quota_micros_min.value === -1;
-      const notRunningInAContainer =
-        node.quota_micros_min.value === null && node.quota_micros_max.value === null;
-      if (limitsNotSet || notRunningInAContainer) {
+      if (limitsNotSet) {
```

Comment:
We can remove the `missingLimits` flag here since we said we would just accept this case even if it's odd but we should probably log some message here about this instead.